### PR TITLE
docs: migrate to github pages

### PR DESCRIPTION
**Pull request by alles-klar**
Migrates all links from readthedocs to github pages.

### CODE REVIEW

**Comment by valentijnscholten** [comment]
thanks, big PR!

**Review by valentijnscholten — CHANGES_REQUESTED** [review]
you don't need a migration to only change a help text. it would save some migrations and the need for others to keep rebasing their migrations. wdyt? (just change the existing migration(s) for this field)

**Comment by alles-klar** [comment]
Changing existing migrations usually not a good idea and can have a lot of side effects. I prefer to use our proved approach, also for small changes.

**Comment by valentijnscholten** [comment]
There are no side affects if it doesn't affect the sql schema.

**Comment by damiencarol** [comment]
I agree, we should remove this hard coded link to the documentation.

**Review by valentijnscholten — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ This checklist is for your information.
 - [ ] Give a meaningful name to your PR, as it may end up being used in the release notes.
 - [ ] Your code is flake8 compliant.
 - [ ] Your code is python 3.6 compliant (specific python >3.6 syntax is currently not accepted).
-- [ ] If this is a new feature and not a bug fix, you've included the proper documentation in the ReadTheDocs documentation folder. https://github.com/DefectDojo/Documentation/tree/master/docs or provide feature documentation in the PR.
+- [ ] If this is a new feature and not a bug fix, you've included the proper documentation in the documentation folder. https://github.com/DefectDojo/django-DefectDojo/tree/dev/docs or provide feature documentation in the PR.
```
Comment (madchap) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:25]: I'd go with just `If this is a new feature and not a bug fix, you've included the proper documentation in the docs at https://github.com/DefectDojo/django-DefectDojo/tree/dev/docs as part of this PR`.
`

**Review by madchap — APPROVED** [review]
Thanks @alles-klar !

**Comment by madchap** [comment]
Removing the `New Migration` label since there is no new migration file.

File: dojo/db_migrations/0001_initial.py [hunk 2]
```diff
@@ -404,7 +404,7 @@ class Migration(migrations.Migration):
             name='JIRA_Conf',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('url', models.URLField(help_text='For configuring Jira, view: https://defectdojo.readthedocs.io/en/latest/features.html#jira-integration', max_length=2000, verbose_name='JIRA URL')),
+                ('url', models.URLField(help_text='For more information how to configuring Jira, read the DefectDojo documentation.', max_length=2000, verbose_name='JIRA URL')),
```
Comment (damiencarol) [inline dojo/db_migrations/0001_initial.py:407]:
```suggestion
                ('url', models.URLField(help_text='For more information how to configure Jira, read the DefectDojo documentation.', max_length=2000, verbose_name='JIRA URL')),
```

**Pull request merged**